automatic/microsoft-windows-terminal/update.ps1

```diff
@@ -1,25 +1,28 @@
 Import-Module au
 
-$releases = 'https://github.com/microsoft/terminal/releases'
+$releases = 'https://api.github.com/repos/microsoft/terminal/releases' # should variable name change?: releases_page instead of releases
```

Comment:
GitHub has a limit on their API usage from one IP. Running this in AppVeyor will trigger their limit fast! Please change this to use an API key.

You should be able to use the variable `$Env:github_api_key` for the API key.

Also please remove the comment. I use $releases consistently in my repo and I would like to continue like that.